pkg/helper/bridges_test.go

```diff
@@ -89,51 +140,54 @@ var _ = Describe("Network desired state bridge parser", func() {
 		BeforeEach(func() {
 			desiredState = empty
 		})
-		It("should return empty map", func() {
+		It("should not be changed", func() {
 			Expect(err).ToNot(HaveOccurred())
-			Expect(obtainedBridgesAndPorts).To(BeEmpty())
+			Expect(updatedDesiredState).To(MatchYAML(desiredState))
 		})
 	})
-	Context("when there is no bridges", func() {
+	Context("when there are no bridges", func() {
 		BeforeEach(func() {
 			desiredState = noBridges
 		})
-		It("should return empty map", func() {
+		It("should not be changed", func() {
 			Expect(err).ToNot(HaveOccurred())
-			Expect(obtainedBridgesAndPorts).To(BeEmpty())
+			Expect(updatedDesiredState).To(MatchYAML(desiredState))
 		})
 	})
 	Context("when there are no bridges up", func() {
 		BeforeEach(func() {
 			desiredState = noBridgesUp
 		})
-		It("should return empty map", func() {
+		It("should not be changed", func() {
 			Expect(err).ToNot(HaveOccurred())
-			Expect(obtainedBridgesAndPorts).To(BeEmpty())
+			Expect(updatedDesiredState).To(MatchYAML(desiredState))
 		})
 	})
 	Context("when there are no ports in the bridge", func() {
 		BeforeEach(func() {
 			desiredState = bridgeWithNoPorts
 		})
-		It("should return the bridge with empty port list", func() {
+		It("should not be changed", func() {
 			Expect(err).ToNot(HaveOccurred())
-			Expect(obtainedBridgesAndPorts).To(HaveKeyWithValue("br1", BeEmpty()))
+			Expect(updatedDesiredState).To(MatchYAML(desiredState))
 		})
 	})
 	Context("when there are bridges up", func() {
 		BeforeEach(func() {
 			desiredState = someBridgesUp
 		})
-		It("should return the map with the bridges and ports", func() {
+		It("should add default vlan filtering to linux-bridge ports", func() {
```

Comment:
I would add another test case for -
`When("some ports have vlan configuration while other don't"`...

Reply:
Added